README.md

```diff
@@ -43,21 +44,47 @@ GET | PROJECT | 125
 Project #125: MyProject
 ```
 
+## Linting Golang
+
+- Requires Node.js (npm) to be installed: <https://nodejs.org/en/download/>
+- Requires Revive to be installed: <https://revive.run/>
+
+```sh
+go get -u github.com/mgechev/revive
+```
+
+```sh
+npm run lint-go
+```
+
 ## Linting markdown
 
-Requires Node.js (npm) to be installed: <https://nodejs.org/en/download/>
+- Requires Node.js (npm) to be installed: <https://nodejs.org/en/download/>
 
 ```sh
 npm install
 
+npm run lint-md
+
+# Some errors can be fixed automatically. Keep in mind that this updates the
+# files in place.
+npm run lint-md-fix
+```
+
+## Linting
+
+You can lint all of the above at the same time by running:
+
+```sh
 npm run lint
 
 # Some errors can be fixed automatically. Keep in mind that this updates the
 # files in place.
 npm run lint-fix
-```
+`
```

Comment:
Seems like a mistake.
```suggestion
```
```

Reply:
Like that? :)

````suggestion
```
````

The trick is that a code block can have more than 3 ticks, as long as it ends with the same number of ticks. So to have a code block contain three ticks, then use 4 ticks:

`````md

Like this:

````suggestion
```
````
`````